js/web/lib/wasm/jsep/webgpu/ops/multihead-attention.ts

```diff
@@ -85,7 +85,7 @@ const validateInputs = (inputs: readonly TensorView[], attributes: AttentionAttr
   let pastSequenceLength = 0;
   let maxSequenceLength = 0;
   const headSize = Math.floor(hiddenSize / attributes.numHeads);
-  if (pastKey && pastValue) {
+  if (pastKey && pastValue && ShapeUtil.size(pastKey.dims) && ShapeUtil.size(pastValue.dims)) {
```

Comment:
Might not need such change.

The original check is fine when past_key has shape like [B, N, P, H] when P === 0 (Need some tests to see whether JS supports such shape).